pkg/apis/sensor/v1alpha1/types.go

```diff
@@ -238,8 +238,8 @@ type ResourceParameter struct {
 	Dest string `json:"dest" protobuf:"bytes,2,name=dest"`
 }
 
-// ResourceParameterSource defines the source for a resource parameter from a event event
-type ResourceParameterSource struct {
+// ParameterSource defines the source for a parameter from a event event
+type ParameterSource struct {
```

Comment:
Should it be `TriggerParameterSource` if we follow the name convention around here?

Reply:
Yup, I will update it.